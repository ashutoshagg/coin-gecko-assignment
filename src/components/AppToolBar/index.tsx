import React from "react";
import { AppBar, Toolbar, Typography } from "@mui/material";

interface IProps {
  name: string | undefined;
}

const AppToolBar = ({ name = "" }: IProps) => {
  return (
    <AppBar position="static">
      <Toolbar
        variant="dense"
        style={{
          display: "flex",
          justifyContent: "center",
        }}
      >
        <Typography variant="h6" color="inherit" component="div">
          {name}
        </Typography>
      </Toolbar>
    </AppBar>
  );
};

export default AppToolBar;
