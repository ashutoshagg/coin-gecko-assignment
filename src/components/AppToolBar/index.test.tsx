import React from "react";
import renderer from "react-test-renderer";
import AppToolBar from "./index";

describe("Components/AppToolBar", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<AppToolBar name="Coin" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
