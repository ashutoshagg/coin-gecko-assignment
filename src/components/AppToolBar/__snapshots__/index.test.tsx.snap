// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppToolBar should match snapshot 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
>
  <div
    className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-10ltm04-MuiToolbar-root"
    style={
      Object {
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <div
      className="MuiTypography-root MuiTypography-h6 css-1qp5wg9-MuiTypography-root"
    >
      Coin
    </div>
  </div>
</header>
`;

exports[`Components/AppToolBar should match snapshot 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
>
  <div
    className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-10ltm04-MuiToolbar-root"
    style={
      Object {
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <div
      className="MuiTypography-root MuiTypography-h6 css-1qp5wg9-MuiTypography-root"
    >
      Coin
    </div>
  </div>
</header>
`;
